internal/testrunner/runners/system/test_config.go

```diff
@@ -58,6 +58,7 @@ type testConfig struct {
 		PidMode           string   `config:"pid_mode"`
 		LinuxCapabilities []string `config:"linux_capabilities"`
 		Runtime           string   `config:"runtime"`
+		ExposePorts       []string `config:"expose_ports"`
```

Comment:
Nit. Have you thought about calling this just `ports`?
```suggestion
		ExposePorts       []string `config:"ports"`
```

Reply:
I wanted to emphasize that these ports are the ones to be exposed. But re-thinking that idea, if it used `ports` as the name in those settings, it looks like is clear that it refers to the ports to be exposed (or forwarded from the host like in VMs).

For instance, docker-compose uses ports: https://docs.docker.com/compose/compose-file/compose-file-v3/#ports
But in Vargrantfile, it is used forwarded_ports: https://developer.hashicorp.com/vagrant/docs/networking/forwarded_ports#defining-a-forwarded-port

I think it could be just "ports".

I'll update the PR

Reply:
Updated in https://github.com/elastic/elastic-package/pull/1795/commits/2af3facfe61da8aa6fc621cc249b252033713e64